ckan/templates/snippets/activities/changed_package.html

```diff
@@ -17,6 +17,8 @@
         <a href="{{ h.url_for('dataset.changes', id=activity.id) }}">
           {{ _('Changes') }}
         </a>
+        <br>
+        Activity ID: {{ activity.id }}
```

Comment:
I'm not clear there is a use need to display an ID and would scare many users. I would have thought the timestamp is enough as a reference?

Reply:
Yup, that's left over from an older version where users chose the range of changes to view using the activity ID. It doesn't need to be there now.